fps/src/main/scala/part1/Chapter3.scala

```diff
@@ -74,10 +77,14 @@ object List:
 
     // Exercise 3.13
     def foldLeftViaFoldRight[A,B](as: List[A], z: B)(f: (B,A) => B): B =
-        foldRight(List.reverse(as), z)((x,y) => f(y,x))
+        foldRight(as, identity[B])((a, g: B=>B) => (b => g(f(b, a))))(z)
```

Comment:
syntax - please write function values with spaces around the double arrow, eg. `f: A => A`